src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -130,7 +129,9 @@ Just set the `pinned` prop.
 				class="app-navigation-entry-icon">
 				<slot v-if="!loading" v-show="isIconShown" name="icon" />
 			</div>
-			<span class="app-navigation-entry__title">{{ title }}</span>
+			<span class="app-navigation-entry__title" :title="title">
```

Comment:
We don't need a title here as there is some actual visible text content with the exact same information. 

https://a11yproject.com/posts/title-attributes/

> Short answer: Don’t use them, except in special circumstances.

Reply:
From side of accessibility, yes, you're right.
Jan's argument to not fully remove it, was for long titles, that get cut and only show the three dots. So they are not visible anymore, and then to be able to see the full text?

Here on forms:
![grafik](https://user-images.githubusercontent.com/47433654/82822250-c2032580-9ea5-11ea-867d-3f39f74db4a7.png)

Reply:
Good point @juliushaertl, but as @jotoeri said the initial reason why we removed it is to offer a way for mouse-users to see the whole title when it’s longer than the width of the navigation.

So to just fix the issue at hand with it appearing _too often_ but not to cut off the entire functionality, I’d say we should merge this. :) We could then separately look into if we can e.g. only show the tooltip if the text overflows in the first place, if possible?